includes/acl/compression/skeleton.h

```diff
@@ -33,6 +33,8 @@
 
 namespace acl
 {
+	static constexpr uint16_t no_parent_bone = 0xFFFF;
```

Comment:
good idea to add a constant, there's a trello card somewhere for this I think. For global constants, I generally prefer to use ALL_CAPS: INVALID_PARENT_BONE_INDEX = 0xFFFF;